libs/vertexai/langchain_google_vertexai/vectorstores/_searcher.py

```diff
@@ -30,17 +31,33 @@ class Searcher(ABC):
     def find_neighbors(
         self,
         embeddings: List[List[float]],
+        sparse_embeddings: Optional[List[Dict[str, List[int] | List[float]]]] = None,
         k: int = 4,
+        rrf_ranking_alpha: float = 1,
         filter_: Union[List[Namespace], None] = None,
         numeric_filter: Union[List[NumericNamespace], None] = None,
-    ) -> List[List[Tuple[str, float]]]:
+    ) -> List[List[Dict[str, Any]]]:
```

Comment:
same here, I'm a little bit concerned we're introducing a breaking change. Is there a chance we could keep it backwards compatible, please?

Reply:
done